web/client/utils/LayersUtils.js

```diff
@@ -64,8 +64,8 @@ const isSupportedLayer = (layer, maptype) => {
     if (layer.type === "mapquest" || layer.type === "bing") {
         return Layers.isSupported(layer.type) && layer.apiKey && layer.apiKey !== "__API_KEY_MAPQUEST__" && !layer.invalid;
     }
-    // type 'ol' represents 'No background' layer
-    if (layer.type === 'ol') {
+    // type 'ol' or 'OpenLayers.Layer' represents 'No background' layer
+    if (layer.type === 'ol' || layer.type === 'OpenLayers.Layer') {
```

Comment:
This is not very MapStore2...ish, can we make the ol / OpenLayers.Layer type something that is handled by the legacy converter and use a better name for the standard type (e.g. empty)

Reply:
is it fine if I parse the type and convert it to **empty** after this [line] ?(https://github.com/geosolutions-it/MapStore2/blob/master/web/client/reducers/config.js#L23)

Reply:
I would do it here: https://github.com/geosolutions-it/MapStore2/blob/master/web/client/utils/ConfigUtils.js#L151